cosmos/x/erc20/keeper/keeper.go

```diff
@@ -47,15 +47,15 @@ func NewKeeper(
 	authority sdk.AccAddress,
 ) *Keeper {
 	return &Keeper{
-		storeKey:   storeKey,
+		StoreKey:   storeKey,
```

Comment:
this should be private.

Reply:
can make it private but have to regenerate everytime